frontend/src/components/Shared/CrossingMapPage/CrossingMapSidebar.js

```diff
@@ -86,14 +117,15 @@ class CrossingMapSidebar extends Component {
       selectedCrossingId,
       selectCrossing,
       currentUser,
-      visibleCrossings,
       allCommunities,
       selectedCrossingName,
       center,
       setSelectedLocationCoordinates,
       setSelectedCommunity,
     } = this.props;
 
+    const nearbyCrossings = this.getNearbyCrossings();
```

Comment:
This seems very expensive to call in render (because we're sorting every crossing). Can we store the result in state or memoize this function?

Reply:
I completely agree. I had started down the path of trying to change it from being called in render to waiting for prop changes and a few other routes. I had considered memoization but hadn't started down the path to implementing it. I've created an issue for it https://github.com/cityofaustin/ctxfloods/issues/155